README.md

```diff
@@ -1,7 +1,7 @@
 # AnalogKeypad
 
 Arduino Analog Keypad library.  
-A keypad library for the [Banggood Analog Keyboard](http://www.banggood.com/AD-Analog-Keyboard-Module-Electronic-Building-Blocks-For-Arduino-p-950742.html).
+A keypad library for the [Banggood Analog Keyboard](https://www.banggood.com/Keypad-4-Button-Key-Module-Switch-Keyboard-For-Arduino-UNO-MEGA2560-Breadboard-p-1532082.html).
```

Comment:
This is a different one than the original it was designed for.  I have already changed the link to the updated and corrected one.  If you want to ALSO include a link to this you will need to get the latest and add this link to it.